Regression in v1.0.1: Windows: `pkg/utils/du_unix.go:37:29: undefined: syscall.Stat_t`

Found in:
- https://github.com/containerd/nerdctl/pull/2642

- - -
 ```
GOOS=windows GOARCH=amd64     make -C /home/runner/work/nerdctl/nerdctl binaries
make[1]: Entering directory '/home/runner/work/nerdctl/nerdctl'
GO111MODULE=on CGO_ENABLED=0 GOOS=windows go build -ldflags "-s -w -X github.com/containerd/nerdctl/pkg/version.Version=6121ddd -X github.com/containerd/nerdctl/pkg/version.Revision=6121dddfdacd0d2aa1c185a7f416d47e89634d2f"   -o /home/runner/work/nerdctl/nerdctl/_output/nerdctl.exe github.com/containerd/nerdctl/cmd/nerdctl
go: downloading github.com/Microsoft/go-winio v0.6.1
go: downloading github.com/Microsoft/hcsshim v0.11.4
go: downloading github.com/inconshreveable/mousetrap v1.1.0
go: downloading github.com/moby/sys/sequential v0.5.0
go: downloading github.com/Azure/go-ansiterm v0.0.0-202106[172](https://github.com/containerd/nerdctl/actions/runs/6929650728/job/18847779599?pr=2642#step:4:173)25240-d185dfc1b5a1
go: downloading go.opencensus.io v0.24.0
go: downloading github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da
# github.com/containerd/accelerated-container-image/pkg/utils
Error: ../../../go/pkg/mod/github.com/containerd/accelerated-container-image@v1.0.1/pkg/utils/du_unix.go:37:29: undefined: syscall.Stat_t
Error: ../../../go/pkg/mod/github.com/containerd/accelerated-container-image@v1.0.1/pkg/utils/du_unix.go:64:31: undefined: syscall.Stat_t
```
https://github.com/containerd/nerdctl/actions/runs/6929650728/job/18847779599?pr=2642

--- a/pkg/utils/du_unix.go
+++ b/pkg/utils/du_unix.go
@@ -1,3 +1,5 @@
+//go:build !windows
+
 /*
    Copyright The Accelerated Container Image Authors
 

